docs/src/pages/custom.md

```diff
@@ -1,2 +1,183 @@
 # Custom Sensitivities
-Coming soon... (you can already add your own sensitivities easily, we just haven't written the documentation yet).
+
+Part of the power of Nabla is its extensibility, specifically in the form of defining
+custom sensitivities for functions.
+This is accomplished by defining methods for `∇` that specialize on the function for
+which you'd like to define sensitivities.
+
+Given a function of the form ``f(x_1, \ldots, x_n)``, we want to be able to compute
+``\frac{\partial f}{\partial x_i}`` for all ``i`` of interest as efficiently as possible.
+Defining our own sensitivities ``\bar{x}_i`` means that ``f`` will be taken as a "unit,"
+and its intermediate operations are not written separately to the tape.
+For more details on that, refer to the [Details](@ref Automatic-Differentiation) section
+of the documentation.
+
+## Intercepting calls
+
+Nabla's approach to RMAD is based on operator overloading.
+Specifically, for each ``x_i`` we wish to differentiate, we need a method for `f` that
+accepts a `Node` in position ``i``.
+There are two primary ways to go about this: `@explicit_intercepts` and `@unionise`.
+
+### `@explicit_intercepts`
+
+When `f` has already been defined, we can extend it to accept `Node`s using this macro.
+
+```@docs
+@explicit_intercepts
+```
+
+As a trivial example, take `sin` for scalar values (not matrix sine).
+We extend it for `Node`s as
+
+```julia
+import Base: sin  # ensure sin can be extended without qualification
+
+@explicit_intercepts sin Tuple{Real}
+```
+
+This generates the following code:
+
+```julia
+begin
+    function sin(##367::Node{<:Real})
+        #= REPL[7]:1 =#
+        Branch(sin, (##367,), getfield(##367, :tape))
+    end
+end
+```
+
+And so calling `sin` with a `Node` argument will produce a `Branch` that holds information
+about the call.
+
+For a nontrivial example, take the `sum` function, which accepts a function argument
+that gets mapped over the input prior to reduction by addition, as well as a `dims`
+keyword argument that permits summing over a subset of the dimensions of the input.
+We want to differentiate with respect to the input array, but not with respect to the
+function argument nor the dimension.
+(Note that Nabla cannot currently differentiate with respect to keyword arguments.)
+We can extend this for `Node`s as
+
+```julia
+import Base: sum
+
+@explicit_intercepts(
+    sum,
+    Tuple{Function, AbstractArray{<:Real}},
+    [false, true],
```

Comment:
this bit i didn't follow. why `[false, true]` here?

Reply:
Good note. Does the most recent commit explain that a bit better?

Reply:
Nice! got it - thanks